frontend/src/edit/hooks/query/useEditQuery.ts

```
@@ -0,0 +1,18 @@
+import { useQuery } from "@tanstack/react-query";
+import type { Ingredient } from "../../types/Ingredient";
+import { useEditApi } from "../api/useEditApi";
+
+export const useEditQuery = () => {
+  const { fetchAllIngredients } = useEditApi();
+  const ingredientQuery = () => {
+    return useQuery<Ingredient[]>({
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)